### PR TITLE
Feature/add beta thompson sampling

### PR DESCRIPTION
**Pull request by f-parfenov**

### CODE REVIEW

File: pulpo/bandits/beta_thompson.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class BetaThompsonBandit(OnlineBandit):
+    def __init__(self, bandit_id: str, arms: List[BetaArm]):
+        super().__init__(bandit_id)
+        self.arms_dict: Dict[str, BetaArm] = {arm.arm_id: arm for arm in arms}
+
+    def choose(self, context: Dict[str, str] = None) -> Arm:
+        arms = list(self.arms_dict.values())
+        return max(arms, key=self._get_score)
```
Comment (pm3310) [inline pulpo/bandits/beta_thompson.py]: Can be simplified `max(self.arms_dict.values(), key=self._get_score)`

File: tests/bandits/beta_thompson.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        assert all([arm.sum_reward == 1 and arm.n == 2 for arm in beta_bandit.arms_dict.values()])
+
+    def test_should_always_select_winner_with_obvious(self):
+        winning_arm = BetaArm("winning_arm", 100000, 99999)
+        losing_arm = BetaArm("losing_arm", 100000, 1)
+
+        beta_bandit = BetaThompsonBandit('my_bandit', [winning_arm, losing_arm])
+
+        for i in range(100):
```
Comment (pm3310) [inline tests/bandits/beta_thompson.py]: can become `for _ in range(100)`

File: tests/bandits/gaussian_thompson.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def test_should_update_an_arm_with_feedback(self):
 
     def test_should_run_for_several_iterations_and_sample_all_arms(self):
         arm_names = ['arm1', 'arm2', 'arm3']
-        arms = [GaussianArm(name, 0.0001, 0, 1) for name in arm_names]
+        arms = [GaussianArm(name, n=5, reward_sum=5, squared_reward_sum=5) for name in arm_names]
```
Comment (pm3310) [inline tests/bandits/gaussian_thompson.py]: 👍

**Comment by pm3310** [comment]
LGTM. 2 minor comments only

File: pulpo/bandits/beta_thompson.py (hunk 1 above)
Comment (f-parfenov) [reply pulpo/bandits/beta_thompson.py]: It's true and it was like that original, I did this split on purpose to make it clearer. What do you think? happy with either

File: tests/bandits/beta_thompson.py (hunk 2 above)
Comment (f-parfenov) [reply tests/bandits/beta_thompson.py]: will correct

**Pull request merged**